CONTRIBUTING.md

```diff
@@ -1,6 +1,21 @@
 # How to Contribute
 
-We're excited you're interested in contributing to Marquez! We'd love your help, and there are plenty of ways to contribute:
+We're excited you're interested in contributing to Marquez! We'd love your help.
+
+We welcome all levels of expertise from novice to experts.
+
+If you are `very new` to the open source contribution:
+* `Fork` this repo
+* `Star` this repo and your own fork
+* In `Issues` tab, read some that interests you
+* You can talk to us directly on `gitter.im` (instructions to join on the project page `README.md`)

```

Comment:
For our Gitter and Twitter acc info, we'll want to mirror what we have in [index.md](https://github.com/MarquezProject/marquez/blob/main/docs/index.md#contributing) under `docs/`. Forgot to add that line here when I made the change :)